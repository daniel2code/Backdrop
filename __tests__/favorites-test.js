import 'react-native';
import React from 'react';
import Favorites from '../src/screens/favorite/index';
import renderer from 'react-test-renderer';

test('Favorites page renders correctly', () => {
  const snap = renderer.create(<Favorites />);

  expect(snap.toJSON()).toMatchSnapshot();
});
