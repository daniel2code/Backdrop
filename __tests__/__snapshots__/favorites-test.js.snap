// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites page renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
      "padding": "6%",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#212227",
        "fontSize": 19,
        "fontWeight": "600",
      }
    }
  >
    Cats I Liked
  </Text>
  <View
    style={
      Object {
        "marginTop": 5,
        "paddingBottom": 15,
      }
    }
  >
    <RCTScrollView
      refreshControl={
        <RefreshControlMock
          onRefresh={[Function]}
          refreshing={false}
        />
      }
    >
      <RCTRefreshControl />
      <View>
        <View
          style={
            Object {
              "flexDirection": "row",
              "flexWrap": "wrap",
              "justifyContent": "space-between",
              "width": "100%",
            }
          }
        />
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
